exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeAdapterTest.java

```diff
@@ -68,32 +68,51 @@ void deployArtifact() throws ServiceLoadingException {
         .build();
     byte[] deploySpec = deployArguments.toByteArray();
 
-    String name = "com.acme:foo:1.2.3";
-    serviceRuntimeAdapter.deployArtifact(name, deploySpec);
+    String name = "com.acme/foo";
+    String version = "1.2.3";
+    ArtifactId artifact = ArtifactId.newBuilder()
+        .setName(name)
+        .setVersion(version)
+        .build();
+    byte[] artifactBytes = artifact.toByteArray();
+
+    serviceRuntimeAdapter.deployArtifact(artifactBytes, deploySpec);
 
-    ServiceArtifactId expectedId = ServiceArtifactId.newJavaId(name);
+    ServiceArtifactId expectedId = ServiceArtifactId.newJavaId(name, version);
     verify(serviceRuntime).deployArtifact(expectedId, artifactFilename);
   }
 
   @Test
   void isArtifactDeployed() {
-    String artifactName = "com.acme:foo:1.3.2";
+    String name = "com.acme/foo";
+    String version = "1.2.3";
+    ArtifactId artifact = ArtifactId.newBuilder()
+        .setName(name)
+        .setVersion(version)
+        .build();
+    byte[] artifactBytes = artifact.toByteArray();
 
-    when(serviceRuntime.isArtifactDeployed(ServiceArtifactId.newJavaId(artifactName)))
+    when(serviceRuntime.isArtifactDeployed(ServiceArtifactId.newJavaId(name, version)))
         .thenReturn(true);
 
-    assertTrue(serviceRuntimeAdapter.isArtifactDeployed(artifactName));
+    assertTrue(serviceRuntimeAdapter.isArtifactDeployed(artifactBytes));
   }
 
   @Test
   void deployArtifactWrongSpec() {
-    String id = "com.acme:foo:1.2.3";
+    String name = "com.acme/foo";
+    String version = "1.2.3";
+    ArtifactId artifact = ArtifactId.newBuilder()
+        .setName(name)
```

Comment:
Here and elsewhere: it does not set runtime id, which is unusual for core. I'd also consider extracting in a method (e.g., createArtifactIdMessage)